### ISSUE TITLE
Please add a way to make "if not this rule

### ISSUE THREAD

**Issue by Jeremy5909** [TodePond/CellPond#365]

**Comment by BestGiter** [comment]
so you could say, if its not a red cell??

**Comment by Jeremy5909** [comment]
Yup and you can do things like "if red is not surrounded by blue, do this"

**Comment by BestGiter** [comment]
maybe it could be with like a thing kindof like the stamp by the triangle, like if you get a circle and like stamp it in a thing, and then it would mean anything but that

**Comment by TeraByte-Official** [comment]
circle stamps are actually a really good idea

**Comment by BestGiter** [comment]
if it was real

![Image](https://github.com/user-attachments/assets/9c8a7c90-aa03-41b8-afb5-0f2f83dc5e40)
![Image](https://github.com/user-attachments/assets/7c8f2bb4-7844-49ff-b516-6424e85cda9a)
![Image](https://github.com/user-attachments/assets/861b9ba3-da97-46a0-bb1d-df7a0345b3ed)

**Comment by Hixie** [comment]
Duplicate of https://github.com/TodePond/CellPond/issues/198
